src/backend/metal/src/device.rs

```diff
@@ -703,6 +703,49 @@ impl Device {
 
         arg
     }
+
+    #[cfg(feature = "core_video")]
```

Comment:
since it doesn't bring any new dependencies, we can expose this unconditionally (with no features)

Reply:
Ok.